### PR TITLE
Bump windows_exporter-package and account for breaking changes

### PR DESCRIPTION
**Pull request by aiyengar2**
Bumps image based on new one created in https://github.com/rancher/windows_exporter-package/pull/3.

https://hub.docker.com/layers/rancher/windows_exporter-package/v0.0.2/images/sha256-725c9e290211066a1456bcc27436c117c42a7024f7f6af4cc552b08c3fa09e89?context=explore

Accounts for breaking changes introduced due to https://github.com/prometheus-community/windows_exporter/pull/667.

Related Issue: https://github.com/rancher/rancher/issues/32519

### CODE REVIEW

File: charts/rancher-monitoring/rancher-monitoring/14.5.101+up14.5.0/charts/windowsExporter/templates/prometheusrule.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  labels: {{ include "windowsExporter.labels" . | nindent 4 }}
+  name: {{ template "windowsExporter.name" . }}
+  namespace: {{ template "windowsExporter.namespace" . }}
+spec:
+  groups:
+  - name: windows-exporter-relabel.rules
+    rules:
+{{- include "windowsExporter.renamedMetricsRules" . | nindent 4 -}}
+{{- end }}{{- end }}
```
[the remark this answers is not in the document]
Comment (aiyengar2) [reply charts/rancher-monitoring/rancher-monitoring/14.5.101+up14.5.0/charts/windowsExporter/templates/prometheusrule.yaml:13]: will add to next commit.

File: charts/rancher-monitoring/rancher-monitoring/14.5.101+up14.5.0/charts/windowsExporter/templates/_helpers.tpl [hunk 2]
```diff
@@ -71,3 +71,43 @@ kubernetes.io/os: windows
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{- define "windowsExporter.renamedMetrics" -}}
+{{- $renamed := dict -}}
+{{/* v0.15.0 */}}
+{{- $_ := set $renamed "windows_mssql_transactions_active_total" "windows_mssql_transactions_active" -}}
```
[the remark this answers is not in the document]
Comment (aiyengar2) [reply charts/rancher-monitoring/rancher-monitoring/14.5.101+up14.5.0/charts/windowsExporter/templates/_helpers.tpl:78]: This was part of a breaking change in windows_exporter.exe v0.15.0: https://github.com/prometheus-community/windows_exporter/releases/tag/v0.15.0

`windows_mssql_transactions_active_total has been removed, and is replaced by windows_mssql_transactions_active`

[the remark this answers is not in the document]
Comment (aiyengar2) [reply charts/rancher-monitoring/rancher-monitoring/14.5.101+up14.5.0/charts/windowsExporter/templates/_helpers.tpl:78]: Previously released in https://github.com/rancher/charts/blob/037989f43e8090a362f86e3ad6a21c981337a0f8/charts/rancher-windows-exporter/rancher-windows-exporter/0.1.000/templates/servicemonitor.yaml#L24-L27

**Review by jiaqiluo — APPROVED** [review]
LGTM.
Rebase is needed.

**Comment by aiyengar2** [comment]
Close in favor of https://github.com/rancher/charts/pull/1274

**Pull request closed without merging**